src/Apps/Artwork/Components/__tests__/PricingContext.test.tsx

```diff
@@ -118,12 +118,36 @@ describe("PricingContext", () => {
     )
   })
 
-  it("displays $0 as the minimum price label is the minimum price is null", async () => {
+  it("displays $0 as the minimum price label if the minimum price is null", async () => {
     const wrapper = await getWrapper()
     expect(wrapper.text()).not.toContain("null")
     expect(wrapper.text()).toContain("$0")
   })
 
+  it("displays $0 as the minimum price label if the minimum price is null", async () => {
+    const wrapper = await getWrapper()
+    expect(wrapper.text()).not.toContain("null")
+    expect(wrapper.text()).toContain("$0")
+  })
+
+  it("displays '1 work' not '0 works' in highlight label if there are zero artworks for the highlighted bin", async () => {
+    const wrapper = await getWrapper()
+    const highlightedBar = wrapper.find(".Bar__BarBox-vx0va3-0").at(2)
+
+    highlightedBar.simulate("mouseenter")
+    expect(wrapper.text()).not.toContain("0 works")
+    expect(wrapper.text()).toContain("1 work")
+  })
+
+  it("displays 'work' singular not 'works' plural in label when there is only 1 artwork in a bin", async () => {
+    const wrapper = await getWrapper()
+    const secondBar = wrapper.find(".Bar__BarBox-vx0va3-0").at(1)
```

Comment:
This selector could be brittle since that little hash at the end is a content hash of the BarBox's css properties, which are just an implementation detail and subject to change.

Luckily `wrapper.find()` is quite flexible. In addition to regular CSS selectors you can use a a react component, or even a react component's `displayName` .  I think `wrapper.find("Bar")` will work in this case because `Bar` is a functional component and therefore the function's `.name` property will be used as the `displayName`.